doc/configuration_options.rst

```diff
@@ -391,3 +391,42 @@ The following options are valid as keys in the ``_config`` dict under
   rosdistro by default.
   To override this, add an entry to this list corresponding to the
   20-default.list file from your forked rosdistro repository.
+
+Specific options in CI build files
+----------------------------------
+
+This yaml file defines the configuration for *CI* jobs.
+
+The file format is specified by the following information:
+
+* ``type: ci-build`` identifies the yaml file as a *CI build file*.
+* ``version: 1`` specifies the specification version of the file.
+
+The following options are valid in version ``1`` (beside the generic options):
+
+* ``build_tool``: the build tool to use.
+  The following are valid values:
+
+  * ``catkin_make_isolated`` (default)
+  * ``colcon``
+
+* ``foundation_packages``: a list of packages which should be installed by
```

Comment:
Maybe use a different name for the option? E.g. `install_packages`.

Reply:
Can we come up with something a little less ambiguous? `default_packages`? `base_package_set`?

Reply:
From an offline conversation, it was decided that `install_packages` should be used.